index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin="" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+    <title>판다마켓</title>
+</head>
+
+<body>
+    <div class="header">
+        <div class="header_contents">
+            <a href="/"><img src="./Group 19.png"></a>
+            <div class="header_login">
+                <a class="link_color" href=".login">로그인
+            </div>
+        </div>
+    </div>
+    <div class="banner">
+        <div class="banner_contents">
+            <div class="banner_looking">
+                <div class="banner_text">일상의 모든 물건을<br>
+                    거래해 보세요</div>
```

Comment:
💊 제안
현재 코드에서는 태그의 여는 태그와 닫는 태그가 잘 정렬되어 있지 않아 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
                <div class="banner_text">
                    일상의 모든 물건을<br>거래해 보세요
                </div>
```